### PR TITLE
EREGCSC-140: Add prototype for Recent Changes box on homepage and in part 433

### PR DESCRIPTION
**Pull request by cgodwin1**
Prototype reuses code and should not be considered the final version.

NOTE: Need URLs before merging!

### CODE REVIEW

File: config/static-assets/static/regulations/css/scss/module/_reg-landing.scss [hunk 1]
```diff
@@ -243,3 +243,53 @@ reg-landing.scss contains styles specific to the regulation landing page
...
+  }
+
+  a {
+    font-family: Open Sans;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 16px;
+    line-height: 24px;
+
+    color: #046791;
```
Comment (theabrad) [inline config/static-assets/static/regulations/css/scss/module/_reg-landing.scss:293]: you probably want to add a class to this anchor tag so the color of the links aren't overwritten.

**Review by theabrad — CHANGES_REQUESTED** [review]

File: config/static-assets/static/regulations/css/scss/module/_custom.scss [hunk 2]
```diff
@@ -1 +1,45 @@
 /* Placeholder for agency-specific modules (i.e. features) */
+
+.recent-changes {
+    margin-top: 30px;
+    width: 100%;
+    height: 175px;
```
Comment (whytheplatypus) [inline config/static-assets/static/regulations/css/scss/module/_custom.scss:6]: On small screens the text is overflowing the box, it looks like changing this to `min-height` fixes things
```suggestion
    min-height: 175px;
```

**Review by whytheplatypus — CHANGES_REQUESTED** [review]

File: guidance/content/regulations/recent-changes.html [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{% comment %}
+    Prototype for recent changes display
+{% endcomment %}
+
+<div class="recent-changes">
+    <h1>Recent Changes: Amended 11/06/2020</h1>
+    <a class="change-link" target="_blank" href="https://www.ecfr.gov/cgi-bin/text-idx?SID=fbe41d98a4ab30061e60b186e69f1b20&mc=true&node=se42.4.433_1400&rgn=div8">
```
Comment (whytheplatypus) [inline guidance/content/regulations/recent-changes.html:7]: I'll work on getting the content up in dev but here's the link for this section: `/433-400/2020-28567#433-400`

```suggestion
    <a class="change-link" target="_blank" href="/433-400/2020-28567#433-400">
```

**Review by whytheplatypus — CHANGES_REQUESTED** [review]

File: guidance/content/regulations/recent-changes.html (hunk 3 above)
Comment (cgodwin1) [reply guidance/content/regulations/recent-changes.html:7]: Thanks!

**Review by whytheplatypus — APPROVED** [review]

**Review by theabrad — APPROVED** [review]

**Pull request merged**